examples/logging/src/main/java/io/opentelemetry/example/logging/LoggingExporterExample.java

```diff
@@ -19,9 +19,7 @@
   public LoggingExporterExample(OpenTelemetry openTelemetry) {
     tracer = openTelemetry.getTracer(INSTRUMENTATION_NAME);
     counter =
-        GlobalMetricsProvider.getMeter(INSTRUMENTATION_NAME)
-            .longCounterBuilder("work_done")
-            .build();
+        GlobalMeterProvider.getMeter(INSTRUMENTATION_NAME).longCounterBuilder("work_done").build();
```

Comment:
Our examples run against the latest release, not against the main branch, so this will get updated after we release 1.1.0. For now, it needs to stay as it was.

Reply:
In fact, I'm quite concerned that the build seems to be passing with this... something might be amiss with our examples check as a part of CI.

Reply:
Hm. This is actually annoyingly tricky to get right. We really want our examples to do 2 things:
1) Verify that we haven't broken anything in the stable APIs and
2) Provide good examples that work against the latest official release (not SNAPSHOT builds).

Unfortunately, I don't think it's possible to meet both of these goals for all the examples, since the stable SDK depends on the unstable metrics, so pulling when we build against snapshots for the stable stuff (to verify that we haven't broken anything), we pull in the unstable metrics version as well. 

Note: this is not your problem to solve in this PR...I'm just clarifying for future us. I'll open an issue with these details so we can make this less confusing. :)

Reply:
But I think the PR cannot be merged without passing all the tests. What can I do, besides leaving the logging example as what it was and removing the user info in the URL?  It seems that further discussion is needed for this conflict to be resolved.

Reply:
I'd like to get #3122 fixed before we merge this, so if you're interested in working on that, that would be great.

Reply:
Note: Now that we've released 1.1.0, this won't be an issue for this PR. Once #3127 has been merged, you can rebase and this PR will be good to go.

Reply:
(#3122 is still an issue, long term, it's just not a blocker for this PR any more. :)

Reply:
Good! I'll do that after #3127 being merged

Reply:
Merged now! Thanks!